tests/playwright/pages/wp-admin-page.js

```diff
@@ -20,20 +18,60 @@ exports.WpAdminPage = class wpAdminPage {
 		}
 
 		await this.page.waitForSelector( 'text=Log In' );
-		await this.page.fill( 'input[name="log"]', 'admin' );
-		await this.page.waitForTimeout( 500 );
-		await this.page.fill( 'input[name="pwd"]', 'password' );
+		await this.page.fill( 'input[name="log"]', this.config.user.username );
+		await this.page.fill( 'input[name="pwd"]', this.config.user.password );
 		await this.page.click( 'text=Log In' );
 		await this.page.waitForSelector( 'text=Dashboard' );
 	}
 
-	async openNewPage() {
-		if ( ! await this.page.$( 'text=Create New Page' ) ) {
-			await this.gotoDashboard();
+	async waitForPanel() {
+		await this.page.waitForSelector( '#elementor-panel-header-title' );
+	}
+
+	async createNewPage() {
+		try {
+			await this.page.click( 'text=Create New Page', { timeout: 5000 } );
+		} catch ( err ) {
+			console.error( "Click on Elementor 'Create New Page' button failed" );
+			await this.page.waitForSelector( 'text=Dashboard' );
+			await this.page.click( 'text=Pages' );
+
+			await Promise.all( [
+				this.page.waitForNavigation( { url: '/wp-admin/post-new.php?post_type=page' } ),
+				this.page.click( 'div[role="main"] >> text=Add New' ),
+			] );
+
+			await Promise.all( [
+				this.page.waitForNavigation(),
+				this.skipTutorial(),
+				this.page.click( 'text=← Back to WordPress Editor Edit with Elementor' ),
+			] );
```

Comment:
Maybe just refine the initial selector (`text=Create New Page`)?
Don't select by text, use some ID or something

Reply:
Handled.